contrib/coccinelle/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docker.io/library/alpine:3.11
+
+LABEL maintainer="maintainer@cilium.io"
+
+RUN apk add curl autoconf automake gcc libc-dev ocaml ocaml-dev ocaml-ocamldoc ocaml-findlib make python bash && \
+    curl -sS -L https://github.com/coccinelle/coccinelle/archive/1.0.8.tar.gz -o coccinelle-1.0.8.tar.gz && \
```

Comment:
Might be worth to declare an `ENV COCCINELL_VERSION 1.0.8` at the top of the `Dockerfile` and use it here and below, so future version bumps are easier.